cmake/config/linux-gcc-clang.txt

```diff
@@ -167,6 +167,11 @@ endif ()
 # Release configuration.
 set (c_compiler_flags_release
     -O3                                                 # optimization level
+    -funroll-loops                                      # Unrolls loops
+    -fgcse-las                                          # Eliminates stores followed by a load to the same positions.
+    -fgcse-sm                                           # Moves load/save instructions out of loops, if possible.
```

Comment:
Do I understand correctly that `-fgcse-lm` is already enabled by `-fgcse` which is automatically enabled thanks to `-O3`?

Reply:
Yes, -fgcse-lm is enabled when -fgcse is enabled, which gets activated automatically by O2, O3 and Os. Therefore, I didn't include it explicitely.